src/components/table/Table.tsx

```diff
@@ -8,7 +8,7 @@ interface ITable {
 
 export function Table({ rows, columns }: ITable) {
   return (
-    <div className="table-container">
+    <div data-testid="table" className="table-container">
```

Comment:
Si agregas este testid usalo para testear que se renderize

Reply:
Now it's being used.